disnake/enums.py

```diff
@@ -603,6 +604,23 @@ def __str__(self) -> str:
         return self.value
 
 
+class StatusDisplayType(Enum):
+    """Specifies an :class:`Activity` display status.
+
+    .. versionadded:: 2.11
+    """
+
+    name = 0  # type: ignore[reportAssignmentType]
```

Comment:
This is a bit unfortunate, since it results in `StatusDisplayType.name` being inferred as `property` (from the base `Enum`) instead of the enum literal, though I'd argue this is a type checker bug[^1].
The only invalid name should be `mro`; `name` and `value` are even officially supported (note at https://docs.python.org/3.13/library/enum.html#enum.property), and work perfectly fine at runtime in cpython enums and our reimplementation.
in any case, it's fine for now, I'd rather see this fixed in type checkers than work around it

[^1]: which also seems to be the position of mypy maintainers, but not pyright